views/accounts/accounts-date.html

```diff
@@ -14,12 +14,13 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-      <form action="" method="post">
-          {% if errorList.length > 0 %}
-          {{ govukErrorSummary({
-          titleText: "There was a problem with the date you gave us",
-          errorList: errorList
-          }) }}
+    <form action="" method="post">
+      {% include "includes/csrf_token.html" %}
+      {% if errorList.length > 0 %}
+      {{ govukErrorSummary({
```

Comment:
Would be good to indent these few lines, since code being changed.

Reply:
I have problems with indentation in the templates when I save the file. I will try to set up my ide to avoid this.

Reply:
Looks better but now lines 22 and 23 need a space indentation too.

Reply:
done